anvil/src/state.rs

```diff
@@ -568,6 +571,7 @@ pub fn post_repaint(
     output: &Output,
     render_element_states: &RenderElementStates,
     space: &Space<WindowElement>,
+    dmabuf_feedback: Option<(&DmabufFeedback, &DmabufFeedback)>,
```

Comment:
These should be separate fields.

Reply:
I don't think using two fields is a good idea, we need both or none. But I agree that without a name it is hard to tell what they are used for. An alternative would be a struct?

Reply:
moved to a separate struct